src/jinacld_tools/aws/services/cloudwatch.py

```diff
@@ -61,34 +69,44 @@ def fetch(self, ec2ARN):
         cloudwatch = boto3.client('cloudwatch', region_name=region,
                 aws_access_key_id=ACCESS_KEY_ID, aws_secret_access_key=SECRET_ACCESS_KEY)
 
-        response = cloudwatch.get_metric_statistics(
-            Namespace='AWS/EC2',
-            Dimensions=[
-                {
-                    'Name': 'instance-id',
-                    'Value': instanceID
-                },
-            ],
-            MetricName='CPUUtilization',
-            StartTime='2021-05-08T23:18:00', #datetime.now() - timedelta(days=7),
-            EndTime='2021-05-09T23:18:00', #datetime.now(),
-            Period=3600, #seconds in one hour
-            Statistics=[
-                'Maximum'
-            ]
-        )
-
-        print(response)
-
-    def store(self):
+        responses = []
+
+        for metric in metrics:
+            response = cloudwatch.get_metric_statistics(
+                Namespace='AWS/EC2',
+                Dimensions=[
+                    {
+                        'Name': 'InstanceId',
+                        'Value': instanceID
+                    },
+                ],
+                MetricName=metric,
+                StartTime=instance.launch_time,
+                EndTime=endTime,
+                Period=3600, #seconds in one hour
+                Statistics=[
+                    'Maximum'
+                ]
+            )
+
+            responses.append(response)
+
+        return responses
+
+    def store(self, metrics):
+        import pickle
+        serializedMetricsObject = pickle.dumps(metrics)
```

Comment:
If `metrics` is json serializable, I would recommend to store it as a json file, rather than pickling it. That way, one can view/download the file from S3 console without needing python